{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "d46820a0_29197ae4",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1096713
      },
      "writtenOn": "2022-12-13T18:13:35Z",
      "side": 1,
      "message": "The \"failure\" we have so far has a ridiculously flaky history, and I\u0027ll bypass if that\u0027s the only \"failure\" we get.",
      "revId": "6f02710ad26f627ca1f1f23dbb190a990b9698f8",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}